### ISSUE TITLE
Doesn't handle .ts files

### ISSUE THREAD

**Issue by billev2k** [boostermedia/clopp#4]
clopp should handle .ts files "out of the box", which is trivial to do, because it is so like .js. However, a way to configure how to process a new file type would be more generic, and highly useful. Perhaps regular expressions to recognize comments, or some such.

**Comment by RikoOphorst** [comment]
You mean something like this?

```javascript
/* #filetype JS */
```

Cause I like that idea :)

Anyway, I'm adding support for TypeScript files right now and I will push a quick fix for that as soon as possible (give me like 30 mins). :)

**Comment by RikoOphorst** [comment]
Hey @billev2k, the fix is live now :) It's version 0.2.0.

You can now enable the `#filetype` tag via the Grunt configuration (disabled by default) like this:

```javascript
        clopp: 
        {
            preprocess: 
            {
                options: {
                    filetypes: true
                },
                files:
                [
                    { src: 'sourcefolder', dest: 'destfolder' }
                ]
            }
        }
```

And in your code you can use this syntax to specify the filetype (it's per file, so it doesn't apply globally to all your project files if your set it in 1 file):

`/* #filetype JS */`

A full list of keywords can be found here: https://www.npmjs.com/package/grunt-clopp#filetype
